lib/svg-sprite/layouter.js

```diff
@@ -141,8 +141,8 @@ SVGSpriteLayouter.prototype = {};
 SVGSpriteLayouter.prototype.layout = function(files, key, mode, cb) {
     this._spriter.info('Laying out «%s» sprite («%s» mode)', key, mode);
     const SVGSpriteLayout = require(`./mode/${mode}`);
-    const config = _.merge(_.merge(_.cloneDeep(defaultConfig[mode]), { svg: this._spriter.config.svg }), this.config[key] || {});
-    const data = _.merge(_.merge(_.merge({}, this._commonData), this._spriter.config.variables), config.variables);
+    const config = merge(merge(merge({}, defaultConfig[mode]), { svg: this._spriter.config.svg }), this.config[key] || {});
```

Comment:
This and the next line could use some cleanup, like splitting them into variables

Reply:
I'll refer this in #458